fetch-runtime/src/index.ts

```diff
@@ -14,7 +14,7 @@ export function createRuntime(methodNames: string[], fileId: Record<'filename',
     runtime[method] = (...args) => {
       let response: Response;
       // TODO: allow customizing URL
-      return fetch('/invoke', {
+      return fetch(process.env.PUBLIC_URL + '/invoke', {
```

Comment:
Do we need this once we support serving from `/invoke`?

Reply:
Perhaps, to leave the possibility of multiple apps on same domain.